src/lib/metricsPrometheus.js

```diff
@@ -34,14 +34,20 @@ const clientRequestsFailureCounter = new client.Counter({
   labelNames: ['clientHost', 'route', 'topic', 'reason'],
```

Comment:
I don't think we need the `clientHost` as tag.

Reply:
this is the same as go client